db/seeds.rb

```diff
@@ -5,3 +5,4 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+28.times { |i| DailyStatistic.create(sum: rand(1000..23000), date: Time.current.to_date+i, movie_id: rand(1..5)) }
```

Comment:
refactor this `Time.current.to_date+i`
this can create records for non existing movies `rand(1..5)`